app/config/config.yml

```diff
@@ -44,12 +44,12 @@ assetic:
     debug:          "%kernel.debug%"
     use_controller: false
     bundles:        [ ]
-    #java: /usr/bin/java
     filters:
         cssrewrite: ~
-        sass:
-            bin:      "/usr/bin/sass"
-            apply_to: "\.scss$"
+        jsqueeze: ~
+        scssphp:
+            # the formatter must be the FQCN (don't use the 'compressed' value)
```

Comment:
What I want is the stack trace of the exception, not the logs (run your assetic dump command in verbose mode to get the stacktrace)